<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Valgrind::Callgrind::ConfigPage</class>
 <widget class="QWidget" name="Valgrind::Callgrind::ConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>244</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cacheSimulation">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify if you want to do full cache simulation. By default, only instruction read accesses will be counted (&amp;quot;Ir&amp;quot;). With cache simulation, further event counters are enabled: &lt;/p&gt;&lt;p&gt;Cache misses on instruction reads (&amp;quot;I1mr&amp;quot;/&amp;quot;ILmr&amp;quot;), &lt;/p&gt;&lt;p&gt;data read accesses (&amp;quot;Dr&amp;quot;) and related cache misses (&amp;quot;D1mr&amp;quot;/&amp;quot;DLmr&amp;quot;), &lt;/p&gt;&lt;p&gt;data write accesses (&amp;quot;Dw&amp;quot;) and related cache misses (&amp;quot;D1mw&amp;quot;/&amp;quot;DLmw&amp;quot;).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Cache simulation</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="branchSimulation">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify if you want to do branch prediction simulation. Further event counters are enabled: Number of executed conditional branches and related predictor misses (&amp;quot;Bc&amp;quot;/&amp;quot;Bcm&amp;quot;), executed indirect jumps and related misses of the jump address predictor (&amp;quot;Bi&amp;quot;/&amp;quot;Bim&amp;quot;).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Branch simulation</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Extra parameters:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="extraParameters">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>callgrind_annotate parameters:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="callgrindAnnotateParameters">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="launchKCachegrind">
        <property name="text">
         <string>Launch KCachegrind after analysis finish</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cacheSimulation</tabstop>
  <tabstop>branchSimulation</tabstop>
  <tabstop>extraParameters</tabstop>
  <tabstop>callgrindAnnotateParameters</tabstop>
  <tabstop>launchKCachegrind</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
